test/converters/test_converters.py

```diff
@@ -617,6 +619,284 @@ def test_integer_to_binary2(self):
             self.assertDictEqual(cst.linear.to_dict(), cst2.linear.to_dict())
             self.assertDictEqual(cst.quadratic.to_dict(), cst2.quadratic.to_dict())
 
+    def test_linear_inequality_to_penalty1(self):
+        """Test special contraint to penalty x+y <= 1 -> P(xy)"""
+
+        op = QuadraticProgram()
+        lip = LinearInequalityToPenalty()
+
+        op.binary_var(name="x")
+        op.binary_var(name="y")
+        # Linear constraints
+        linear_constraint = {"x": 1, "y": 1}
+        op.linear_constraint(linear_constraint, Constraint.Sense.LE, 1, "P(xy)")
```

Comment:
FYI: you can simply use strings of equalities
```
op.linear_constraint(linear_constraint, '<=', 1, "P(xy)")
```

Reply:
We will leave the equality as is. May change when we refactor test cases.